apps/src/lib/node/ledger/shell/mod.rs

```diff
@@ -158,6 +158,22 @@ pub fn reset(config: config::Ledger) -> Result<()> {
     Ok(())
 }
 
+pub fn rollback(config: config::Ledger) -> Result<()> {
+    // Rollback Tendermint state
+    tracing::info!("Rollback Tendermint state");
```

Comment:
Should we do a backup here before reverting the state in case anything went wrong? The Tendermint rollback command doesn't look to be doing so. We could also just ask the users to do the backup themselves before running this command